src/lib/tvgLoadModule.h

```diff
@@ -36,7 +36,7 @@ class LoadModule
     float vy = 0;
     float vw = 0;
     float vh = 0;
-    float w = 0, h = 0;         //default image size
+    float w = 1.0f, h = 1.0f;         //default image size
```

Comment:
You should not change the base policy for SVG. If necessary, please arrange the default values in SvgLoader.

Reply:
done